pkgs/build-support/bintools-wrapper/default.nix

```diff
@@ -164,6 +164,20 @@ stdenv.mkDerivation {
       set +u
     '';
 
+  emulation = let
```

Comment:
I think I will move this to the lib, with some of the other misc per-tool stuff / provide a way for `{build,host,target}Platform` (e.g. via `crossConfig`) to override it. That, however, is a non-hash-breaking change so no reason to rush it now.